src/components/shell/shell.tsx

```diff
@@ -14,7 +14,10 @@ import {
  * @slot panel-start - A slot for adding the starting `calcite-shell-panel`.
  * @slot panel-end - A slot for adding the ending `calcite-shell-panel`.
  * @slot center-row - A slot for adding content to the center row.
+ * @slot modal - A slot for adding a `calcite-modal`. When placed in this slot, the modal position will be constrained to the extent of the shell.
```

Comment:
Yeah, they should be consistent one way or the other. I had used `alerts` since we kind of incentivize multiple with the `queued` display and only one `modal` is shown to a user at once, but I'll make them both plural.

Reply:
@driskull - updated to `modals` - going to merge.